src/components/Header/index.jsx

```diff
@@ -373,13 +369,9 @@ const Header = () => {
                 <Dropdown.Item onClick={() => logout(null)}>
                   Log out
                 </Dropdown.Item>
-                <Dropdown.Divider />
               </Dropdown.Menu>
             ) : (
-              <Dropdown.Menu>
-                <Dropdown.Item onClick={handleLogin}>Log in</Dropdown.Item>
-                <Dropdown.Divider />
-              </Dropdown.Menu>
```

Comment:
Can you please explain the need to remove the login button?

Reply:
As explained above, the button is redundant. Earlier the dropdown had some purpose as the Light to Dark theme switch was located in that drop down but now it only houses a redundant button.
Replacing the `<Dropdown.Menu/>` with an empty `<div />` removes any drop down when the user is not logged in.